streamlit-ui/src/app.py

```diff
@@ -0,0 +1,75 @@
+
+import jwt
+import streamlit as st
+
+from src import iou
+from src import config
```

Comment:
For me (with venv) I need this to be

```suggestion
from python.src import iou
from python.src import config
```

But I'm not sure exactly how this works (yet)

Reply:
works in docker without the python. And why python?? I'm lost

Reply:
As discussed, this does not seem to be quite right, as it's certainly not the `python.src` thing we want to import 🤣